main.py

```diff
@@ -62,8 +62,34 @@ def h_run(args):
 		print("Failed to start: No halibot configuration found in the current directory!")
 		return
 
-	logging.basicConfig(level=logging.DEBUG)
 	bot = halibot.Halibot()
+	bot._load_config()
+
+	logfile = None
+	loglevel = logging.DEBUG
+	loglevel_str = "DEBUG"
+	loglevels = { "CRITICAL": logging.CRITICAL, "ERROR": logging.ERROR, "WARNING": logging.WARNING, "INFO": logging.INFO,
+			"DEBUG": logging.DEBUG, "NOTSET": logging.NOTSET }
```

Comment:
Might be easier on the eyes to have these on separate lines rather than two longish ones